connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkSsaiDescription.kt

```diff
@@ -0,0 +1,89 @@
+package com.theoplayer.android.connector.uplynk
+
+import com.theoplayer.android.api.source.ssai.CustomSsaiDescription
+import kotlinx.serialization.Serializable
+
+/**
+ * The configuration for server-side ad insertion using the [UplynkConnector].
+ */
+@Serializable
+data class UplynkSsaiDescription(
+    val prefix: String? = null,
+    val assetIds: List<String> = listOf(),
+    val externalId: List<String> = listOf(),
+    val userId: String? = null,
+    val preplayParameters: LinkedHashMap<String, String> = linkedMapOf()
+): CustomSsaiDescription() {
+
+    override val customIntegration: String
+        get() = UplynkConnector.INTEGRATION_ID
+
+    /**
+     * A builder for a [UplynkSsaiDescription].
+     */
+    class Builder {
+        private var prefix: String? = null
+        /**
+         * Sets the prefix to use for Uplynk Media Platform Preplay API and Asset Info API requests.
+         *
+         * <ul>
+         *     <li>If no prefix is set the default origin is used: https://content.uplynk.com
+         * </ul>
```

Comment:
Nit-pick: with Dokka, you can use Markdown-style lists instead of HTML:
```suggestion
         * - If no prefix is set the default origin is used: https://content.uplynk.com
```